feature-libs/customer-ticketing/components/details/customer-ticketing-messages/customer-ticketing-messages.component.spec.ts

```diff
@@ -68,9 +66,12 @@ describe('CustomerTicketMessagesComponent', () => {
     spyOn(customerTicketingFacade, 'createTicketEvent').and.callThrough();
     component.onSend(mockSendEvent);
 
-    expect(customerTicketingFacade.createTicketEvent).toHaveBeenCalledWith({
-      message: 'mock message',
-    });
+    expect(customerTicketingFacade.createTicketEvent).toHaveBeenCalledWith(
+      {
+        message: 'mock message',
+      },
+      false
```

Comment:
What does the second parameter mean?
for understanding purposes can a const be created:
const mustWaitForAttachment = false;  (it could also be includesAttachment)
```suggestion
      mustWaitForAttachment
```